Sources/MessagingInApp/MessagingInAppImplementation.swift

```diff
@@ -99,13 +99,10 @@ extension MessagingInAppImplementation: GistDelegate {
         logger.debug("in-app action made. \(action), \(message.describeForLogs)")
 
         // a close action does not count as a clicked action.
-        guard action != "gist://close" else {
-            return
-        }
-
-        if let deliveryId = getDeliveryId(from: message) {
-            // the state of the SDK does not change if adding this queue task isn't successful so ignore result
-            _ = queue.addTrackInAppDeliveryTask(deliveryId: deliveryId, event: .clicked)
+        if action != "gist://close" {
+            if let deliveryId = getDeliveryId(from: message) {
+                _ = queue.addTrackInAppDeliveryTask(deliveryId: deliveryId, event: .clicked)
```

Comment:
```suggestion
                // the state of the SDK does not change if adding this queue task isn't successful so ignore result
                _ = queue.addTrackInAppDeliveryTask(deliveryId: deliveryId, event: .clicked)
```

I suggest we keep this comment as it explains the decision of ignore the `queue.` result.